ZelBack/src/lib/server.js

```diff
@@ -1,14 +1,15 @@
 const express = require('express');
 const eWS = require('express-ws');
-const bodyParser = require('body-parser');
+// const bodyParser = require('body-parser');
 const cors = require('cors');
 const morgan = require('morgan');
 
 const expressWs = eWS(express());
 const { app } = expressWs;
 
 app.use(morgan('combined'));
-app.use(bodyParser.json());
+app.use(express.json());
```

Comment:
ok our package ensure that express is on version 4.18 so this will work. There should not be any fluxos version with lower express than 4.16.